code/addons/storyshots/storyshots-core/src/frameworks/angular/loader.ts

```diff
@@ -33,17 +33,35 @@ function test(options: StoryshotsOptions): boolean {
 function load(options: StoryshotsOptions) {
   setupAngularJestPreset();
 
-  const storybook = jest.requireActual('@storybook/angular');
-  const clientAPI = jest.requireActual('@storybook/client-api');
+  let mockStartedAPI: any;
 
-  const api = {
-    ...clientAPI,
-    ...storybook,
-  };
+  jest.mock('@storybook/core-client', () => {
+    const coreClientAPI = jest.requireActual('@storybook/core-client');
+
+    return {
+      ...coreClientAPI,
+      start: (...args: any[]) => {
+        mockStartedAPI = coreClientAPI.start(...args);
+        return mockStartedAPI;
+      },
+    };
+  });
+
+  jest.mock('@storybook/angular', () => {
+    const renderAPI = jest.requireActual('@storybook/angular');
+
+    renderAPI.addDecorator = mockStartedAPI.clientApi.addDecorator;
+    renderAPI.addParameters = mockStartedAPI.clientApi.addParameters;
```

Comment:
Is this just to re-add the two APIs that were dropped? Perhaps we should discuss what's going wrong here.

Reply:
Yeah, I'm adding the APIS that got deprecated/removed to storyshots because storyshots still needs it :(

Reply:
Shouldn't it be possible for SS to not use those APIs? That might be easier than all this.

Reply:
I didn't know how to, Do you have any ideas?

Reply:
Hmm, have you tested this PR in SSv6? I think you are going to have the same issue due to https://github.com/storybookjs/storybook/blob/9a07fcba8a581558d0fa8262989ef83b70a507a0/code/lib/builder-webpack5/templates/virtualModuleEntry.template.js

Reply:
client-api still exports it, it's the renderer package that we expect users to import where I really wanted it gone.